CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 5.1.1
+
+## Remove obsolete documentation
+
+Remove the obsolete docs on `Running Tests` and `Debugging Tests`.
```

Comment:
If you want, just remove the release notes for this? It does not necessarily require a release. Then we can merge it,